### DISCUSSION TITLE
Evaluating single point values from a tensor plot

### DISCUSSION THREAD

**Discussion by mwaseemashraf** [mtex-toolbox/mtex#1879] [Ask Anything] [answered]
Hi,

I have a tensor plot using the overloaded plot function; plot(tensorObject). How do I extract single-point values from it?

Thanks

**Comment by ralfHielscher** [comment, marked as answer]
what about
```matlab
v = vector3d(1,0,0)
tensorObject.directionalMagnitude(v)
```
Ralf.

**Reply by mwaseemashraf** [reply]
Thank you for the response. Additionally, is there any way I can knit the single values (as evaluated in your answer) from different tensor plots into a single one (tensor plot) using mtex. I can code it, I am just curious if Mtex has any such function.

**Reply by ralfHielscher** [reply]
I'm sorry, I do not understand what kind of graphics you want to have.
